test/e2e/metrics_test.go

```diff
@@ -57,6 +60,23 @@ func (s *MetricsSuite) TestMetricsEndpoint() {
 	})
 }
 
+func (s *MetricsSuite) TestRetryMetrics() {
```

Comment:
For now, the e2e test is added to the `MetricsSuite`(within `api` test group) since we don't really have a suite for testing metrics functionalities.